DESCRIPTION

```diff
@@ -1,20 +1,37 @@
-Package: pacta.r.package
-Title: What the Package Does (One Line, Title Case)
+Package: pacta.loanbook
+Title: Umbrella package for all PACTA for Banks packages and documentation
 Version: 0.0.0.9000
 Authors@R: 
-    c(person(given = "First", 
-             family = "Last", 
-             role = c("aut", "cre"),
-             email = "first.last@example.com", 
-             comment = c(ORCID = "YOUR-ORCID-ID")),
-      person(given = "Rocky Mountain Institute", 
-             role = c("cph", "fnd"),
-             email = "PACTA4investors@rmi.org"))
-Description: What the package does (one paragraph).
+    c(
+      person(
+        given = "Jacob",
+        family = "Kastl",
+        role = c("aut", "cre", "ctr"),
+        email = "jacob.kastl@gmail.com",
+        comment = c(ORCID = "0009-0000-8281-8129")
+      ),
+      person(
+        given = "Jackson",
+        family = "Hoffart",
+        role = c("aut", "ctr"),
+        email = "jackson.hoffart@gmail.com",
+        comment = c(ORCID = "0000-0002-8600-5042")
+      ),
+      person(
+        given = "RMI",
+        role = c("cph", "fnd"),
+        email = "PACTA4banks@rmi.org"
+      )
+    )
+Description: Installs all PACTA for Banks packages needed for a standard PACTA
+  for Banks analysis of a corporate loan book. It also provides thorough
+  documentation - the PACTA for Banks cookbook - on how to run a PACTA for Banks
+  analysis. This includes prerequisites for the analysis, the separate steps of
+  the analysis, interpretation of the results, and advanced use cases.
```

Comment:
NIT (reworded a bit), not suggesting a code commit since I can't format it properly for the DESCRIPTION file.

> PACTA for Banks is a tool that allows banks to calculate the climate alignment of their corporate lending portfolios. This package is designed to make it easy to install and load multiple PACTA for Banks packages in a single step. It also provides ....

Reply:
addressed